cluster-autoscaler/cloudprovider/mcm/mcm_cloud_provider_test.go

```diff
@@ -341,12 +349,41 @@ func TestRefresh(t *testing.T) {
 	}
 	table := []data{
 		{
+
+			"should set available replicas of mcm as zero",
```

Comment:
```suggestion
			"should return an error if MCM has zero available replicas",
```
Here, you write the description of the test. The test does not set MCM replicas. It checks the replicas and takes action accordingly